service/src/main/kotlin/io/provenance/api/frameworks/web/config/WebConfig.kt

```diff
@@ -4,17 +4,29 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import org.springframework.context.annotation.Configuration
 import org.springframework.http.codec.ServerCodecConfigurer
 import org.springframework.http.codec.json.Jackson2JsonEncoder
+import org.springframework.http.codec.multipart.DefaultPartHttpMessageReader
+import org.springframework.http.codec.multipart.MultipartHttpMessageReader
 import org.springframework.web.reactive.config.EnableWebFlux
 import org.springframework.web.reactive.config.WebFluxConfigurer
 
 @Configuration
 @EnableWebFlux
 class WebConfig(
-    private val objectMapper: ObjectMapper
+    private val objectMapper: ObjectMapper,
 ) : WebFluxConfigurer {
 
     override fun configureHttpMessageCodecs(configurer: ServerCodecConfigurer) {
         // Enable support for serializing protos to JSON in API responses
         configurer.defaultCodecs().jackson2JsonEncoder(Jackson2JsonEncoder(objectMapper))
+        configurer.defaultCodecs().maxInMemorySize(500 * 1024 * 1024)
+
+        val partReader = DefaultPartHttpMessageReader()
+        partReader.setMaxHeadersSize(16 * 1024 * 1024)
```

Comment:
setting this due to https://rollbar.com/figure/p8e-cee-api/items/711/?person_page=0&#traceback

Reply:
For those outside of Figure, the linked error says
> Part headers exceeded the memory usage limit of 8192 bytes